PlaceRouteHierFlow/placer/ILP_solver.cpp

```diff
@@ -1980,6 +1980,35 @@ bool ILP_solver::FrameSolveILPSymphony(const design& mydesign, const SeqPair& cu
     }
   }
 
+  // matchblock
```

Comment:
Out of curiosity, what does the distance between blocks have to do with the sequence pairs? Is the snippet below setting scaling factors for these objects to be used later in ILP?

Reply:
Distance between blocks do not have relation with sequence pair

Reply:
Then, what is the code below expected to do?

Reply:
@soneryaldiz the sequence pair is used to determine order of the blocks to find the sign for subtraction.
For place closer, the objective of ILP tries to reduce distance between the blocks.
For example A and B, if they appear on the sequence pair as A to the left of B, then the objective has to be B<sub>x</sub> - A<sub>x</sub>. If it is the other way then, the objective should have A<sub>x</sub> - B<sub>x</sub>. This is what the code is trying to do.